Misc/NEWS.d/next/Library/2024-05-16-00-08-53.gh-issue-119067.1v2ny6.rst

```diff
@@ -0,0 +1,3 @@
+Add :const:`visible` to :mod:`string`, containing all non-blank ASCII characters
+
+(i.e. everything in :const:`printable` except for :const:`whitespace`).

```

Comment:
```suggestion
Add :const:`visible` to :mod:`string`, containing all non-blank ASCII characters
(i.e., everything in :const:`printable` except for :const:`whitespace`).
```

I would suggest adding a WhatsNew entry as well since it is could be useful for users.